drift-transport-netty/src/test/java/com/facebook/drift/transport/netty/codec/TestHeaderTransport.java

```diff
@@ -73,17 +122,25 @@ private static void assertDecodeFrameInfo(ByteBuf message, Optional<FrameInfo> f
         }
     }
 
-    private static ByteBuf createTestFrame(ByteBufAllocator allocator, String methodName, byte messageType, int sequenceId, Protocol protocol, boolean supportOutOfOrderResponse)
+    private static ByteBuf createTestFrame(
+            ByteBufAllocator allocator,
+            String methodName,
+            byte messageType,
+            int sequenceId,
+            Protocol protocol,
+            boolean supportOutOfOrderResponse,
+            List<Transform> transforms)
             throws TException
     {
         ThriftFrame frame = new ThriftFrame(
                 sequenceId,
                 createTestMessage(allocator, methodName, messageType, sequenceId, protocol),
                 ImmutableMap.of("header", "value"),
+                transforms,
                 HEADER,
                 protocol,
                 supportOutOfOrderResponse);
-        return HeaderTransport.encodeFrame(frame);
+        return encodeFrame(frame);
```

Comment:
Please improve this test by writing something non trivial in the `createTestMessage` to make sure that the message is large enough.